frontend/projects/upgrade/src/app/core/segments/store/segments.model.ts

```diff
@@ -1,6 +1,11 @@
 import { AppState } from '../../core.state';
 import { EntityState } from '@ngrx/entity';
 import { SEGMENT_TYPE, SEGMENT_STATUS } from 'upgrade_types';
+import {
+  SEGMENT_SEARCH_KEY,
+  SEGMENT_SORT_AS,
+  SEGMENT_SORT_KEY,
+} from '../../../../../../../../types/src/Experiment/enums';
```

Comment:
For consistency, I think we can import these three from 'upgrade_types' and then export them, so that everything's coming from the model, like it does with experiments. We assume that the `types/` directory is copied from the top level into frontend/packages/Upgrade/ as part of every build.